bin/foreman-tasks

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+foreman_root = '/usr/share/foreman'
```

Comment:
I guess this can stay hardcoded for now, as core is only supported on Linux, where that path is always correct.